File: base/src/main/java/com/cangwang/base/api/WebApi.java
package com.cangwang.base.api;

import com.cangwang.core.MBaseApi;

/**
 * Created by cangwang on 2018/2/6.
 */

public interface WebApi extends MBaseApi{
    void loadWeb(String url,String title);
    void removeWeb();
}

File: base/src/main/java/com/cangwang/base/api/WebApi.kt
package com.cangwang.base.api

import com.cangwang.core.MBaseApi

/**
 * Created by cangwang on 2018/2/6.
 */
interface WebApi : MBaseApi {
    fun loadWeb(url: String?, title: String?)
    fun removeWeb()
}